src/licensedcode/data/rules/gpl-2.0-plus_with_bison-exception-2.2_3.yml

```diff
@@ -0,0 +1,4 @@
+license_expression: gpl-2.0-plus WITH bison-exception-2.2
```

Comment:
This should instead:
```
license_expression: bsd-new AND gpl-2.0-plus WITH bison-exception-2.2
 is_license_notice: yes
minimum_coverage: 99
referenced_filenames:
    - Copyright.txt
```

You may also want to create a second rule with only the BSD parts:
```
 Distributed under the OSI-approved BSD License (the "License");
  see accompanying file Copyright.txt for details.
```

Reply:
Right! Updating.

Reply:
So I've created a second rule with the BSD parts beforehand only, in this [commit]() but didn't add  `bsd-new AND` to the `license_expression` of `gpl-2.0-plus WITH bison-exception-2.2`.